docs/source/rst_quick_ref.rst

```diff
@@ -0,0 +1,449 @@
+                Do something.
+                :Parameters:
+                    size
+                        The size of the fox (in meters)
+                :rtype: str
+                :return: The number of foxes
+                '''
+                pass
+
+     - .. code:: python
+
+        def hey(**kargs):
+            '''
+            Do something.
+            :Parameters:
+                size
+                    The size of the fox (in meters)
+            :rtype: str
+            :return: The number of foxes
+            '''
+            pass
+
+   * - ::
+    
+        .. python:: 
+
+            def hey(**kargs):
+                '''
+                Do something.
+                :Parameters:
+                    size
+                        The size of the fox (in meters)
+                :rtype: str
+                :return: The number of foxes
+                '''
+                pass
+
+     - .. python:: 
+
+        def hey(**kargs):
+            '''
+            Do something.
+            :Parameters:
+                size
+                    The size of the fox (in meters)
+            :rtype: str
+            :return: The number of foxes
+            '''
+            pass
+
+math
+++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+     
+   * - :: 
+
+        .. math::
+
+        α_t(i) = P(O_1, O_2, … O_t, q_t = S_i λ)
+
+     - .. math::
+
+        α_t(i) = P(O_1, O_2, … O_t, q_t = S_i λ)
+
+raw
++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+     
+   * - :: 
+
+        .. raw:: html
+
+            <hr />
+
+     - .. raw:: html
+
+            <hr />
+
+figure & image
+++++++++++++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+     
+   * - :: 
+
+        .. figure:: https://khms1.googleapis.com/kh?v=878&hl=en-US&x=2273&y=3006&z=13
+
+            This is the caption of the figure (a simple paragraph).
+
+            The legend consists of all elements after the caption.  In this
+            case, the legend consists of this paragraph.
+
+     - .. figure:: https://khms1.googleapis.com/kh?v=878&hl=en-US&x=2273&y=3006&z=13
+
+            This is the caption of the figure (a simple paragraph).
+
+            The legend consists of all elements after the caption.  In this
+            case, the legend consists of this paragraph.
+
+tables
+++++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+     
+   * - ::
+
+            .. table:: Truth table for "not"
+                :widths: auto
+
+                =====  =====
+                A      not A
+                =====  =====
+                False  True
+                True   False
+                =====  =====
+    
+     -  .. table:: Truth table for "not"
+            :widths: auto
+
+            =====  =====
+            A      not A
+            =====  =====
+            False  True
+            True   False
+            =====  =====
```

Comment:
In Chromium the "not A" column header is spread over two lines, which looks really bad. The problem might be due to missing CSS: there are a number of classes with a `rst-` prefix defined on this table, but no loaded style sheet defines those.

Reply:
Same here on safari